tools/policy-gen/bootstrap/root.go

```diff
@@ -110,13 +110,34 @@ func generateProto(c config) error {
 	if err != nil {
 		return err
 	}
-	return validatorTemplate.Execute(f, map[string]interface{}{
+
+	err = validatorTemplate.Execute(f, map[string]interface{}{
 		"name":              c.name,
 		"version":           c.version,
 		"generateTargetRef": c.generateTargetRef,
 		"generateTo":        c.generateTo,
 		"generateFrom":      c.generateFrom,
 	})
+	if err != nil {
+		return err
+	}
+
+	f, err = os.Create(path.Join(apiPath, c.lowercase()+"_helpers.go"))
```

Comment:
If this file is meant to be auto generated and not modified putt zz_generated in the name would help identify it!

Reply:
No, this is the same as `validator.go`. It should be generated only as a bootstrap, people can add other helper functions to it. So it's not updated during `make generate`

Reply:
But aren't these ones always required to be created? Shouldn't they be just generated code? Ok people could add other helpers but these ones seem just as required as TypeDescriptor for example

Reply:
In that case, we have to pull the information about the existence of `to/from` arrays into `KumaPolicyOptions`. Otherwise, during the `make generate` we don't know if helpers should be generated

Reply:
Don't you have the name of the fields in your object in the protoc-gen-kumapolicy? Sounds like you should be able to deduce this no? Like you do the `zz_generated.deepcopy.pb.go` and the `zz_generated.resource.go` ?

Reply:
With message.Fields for .e.g?

Reply:
Right, I didn't realize it's pretty easy to get with `protogen` package. Fixed!